src/components/manage/index.js

```diff
@@ -519,7 +524,7 @@ export class Manage extends Component {
     }
 
     const tierStartAndEndTime = (tier, index) => {
-      const disabled = !this.state.ownerCurrentUser || !tier.updatable || crowdsaleHasEnded
+      const disabled = !this.state.ownerCurrentUser || !tier.updatable || this.tierHasStarted(index) || crowdsaleHasEnded
```

Comment:
End time should be editable if crowdsale is started and not finished yet (if tier is editable)

Reply:
Mh! I'll need #586 to be merged before fixing this, as fixes in that PR are required for this change.

Reply:
change done